firmware/hw_layer/debounce.cpp

```diff
@@ -9,6 +9,7 @@
 #include "pin_repository.h"
 #include "engine_configuration.h"
 #include "hardware.h"
+#include "hal.h"
```

Comment:
this does not look right. no one is including hal.h why would debounce.cpp?!

Reply:
brute force programming. it's removed now

Reply:
I am not the one to criticize brute force programming.